### DISCUSSION TITLE
RouterView Transition to same page but with different props

### DISCUSSION THREAD

**Discussion by Jazcash** [posva/unplugin-vue-router#146] [Questions and Help] [answered]
https://codesandbox.io/p/sandbox/vue-router-suspense-transition-teleport-bug-forked-ens0xr?file=%252Fsrc%252FApp.vue

In the minimal example above, switching between routes has a fade transition. But switching between B and C has no transition, because they're technically the same view (`[id].vue`). Is there a solution for this?

**Comment by posva** [comment, marked as answer]
https://router.vuejs.org/guide/advanced/transitions.html#forcing-a-transition-between-reused-views
